testdata/tfe_config/fogg.yml

```diff
@@ -0,0 +1,29 @@
+defaults:
+  backend:
+    kind: remote
+    host_name: si.prod.tfe.czi.technology
+    organization: shared-infra
+  owner: foo@example.com
+  project: foo
+  providers: {}
+  terraform_version: 1.1.1
+envs:
+  comp1: {}
+  comp2: {}
+accounts:
+  account: {}
+tfe:
+  terraform_version: 1.2.6
+  gh_org: chanzuckerberg
+  gh_repo: fogg
+  tfe_org: si.prod.tfe.czi.technology
+  module_source: github.com/chanzuckerberg/cztack//aws-params-reader-policy?ref=v0.15.1
```

Comment:
in the real world, you would use something like "git@github.com:chanzuckerberg/shared-infra//terraform/modules/tfe-management?ref=tfe-management-v1.0.0"

Reply:
I had to change it so CICD would pass since its a private module